pkg/operator/controller/ingress/deployment.go

```diff
@@ -1690,3 +1701,14 @@ func capReloadIntervalValue(interval time.Duration) time.Duration {
 		return interval
 	}
 }
+
+// SingleReplica determines if ingress controllers are deployed in SingleReplica topology
```

Comment:
```suggestion
// singleReplica determines if ingress controllers are deployed in SingleReplica topology
```
But this is trivial enough that it shouldn't hold up the PR.  I see nothing has changed other than some comments and rebases since the last /lgtm, so
/lgtm